analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -322,7 +322,12 @@ private class Graph(private val nodeMap: MutableMap<Identifier, Set<Identifier>>
 
 // See https://golang.org/ref/mod#pseudo-versions.
 // Format for no known base version, e.g. v0.0.0-20191109021931-daa7c04131f5:
-private val PSEUDO_VERSION_REGEX = "^v0\\.0\\.0-[\\d]{14}-(?<sha1>[0-9a-f]+)$".toRegex()
+private val PSEUDO_VERSION_REGEXES = listOf(
+    // no known base version, e.g. v0.0.0-20191109021931-daa7c04131f5.
```

Comment:
Shouldn't the comment from line 324 now be dropped? And these comments start with "Format for"?